src/ActionList/ActionList.stories.tsx

```diff
@@ -242,34 +242,53 @@ LinkItemPlayground.argTypes = {
   },
 }
 
-export const GroupPlayground: Story<ActionListGroupProps> = args => (
-  <ActionList>
-    <ActionList.Group {...args}>
-      <ActionList.Item>Item 1</ActionList.Item>
-      <ActionList.Item>Item 2</ActionList.Item>
-    </ActionList.Group>
+type HeadingProps = {
+  headingVariant?: 'subtle' | 'filled'
+}
+
+export const HeadingPlayground: Story<ActionListProps & HeadingProps> = args => (
+  <ActionList headingProps={{title: 'Group title', variant: args.headingVariant}} {...args}>
+    <ActionList.Item>Copy link</ActionList.Item>
+    <ActionList.Item>Quote reply</ActionList.Item>
+    <ActionList.Item>Edit comment</ActionList.Item>
   </ActionList>
```

Comment:
```suggestion
export const HeadingPlayground: Story<ActionListProps & HeadingProps> = args => (
  <ActionList headingProps={{title: args.title, variant: args.variant, subtitle: args.subTitle, headingLevel: args.headingLevel}} {...args}>
  </ActionList>
```

If it doesn't work to just render the empty list that's fine, it just looks cleaner as this story is only focused on the heading.

Reply:
That should work. And it gets around the fact that variant is overloaded.